package.json

```diff
@@ -50,7 +51,8 @@
     "prettier": "^2.4.1",
     "shuffle-array": "^1.0.1",
     "ts-jest": "^26.4.4",
-    "typescript": "^4.1.3"
+    "typescript": "^4.1.3",
+    "standard-version": "^9.3.2"
```

Comment:
nit: move `standard-version` up so that deps are in alphabetical order.

It's preferable to use `yarn add` instead of manually adding dependencies to this file since that keeps things in order. Not a huge deal since package.json is simple enough to be edited manually.

Out of order dependencies make it slightly harder for people to figure out the version of our dependencies, and also they make future PR diffs look like they are modifying several dependencies, but in reality they are just re-ordering dependencies to the correct order because they used `yarn add`.